biggus/__init__.py

```diff
@@ -2287,6 +2346,103 @@ def _sliced_shape(shape, keys):
     return sliced_shape
 
 
+class broadcast(object):
+    def __init__(self, array1, array2):
+        self._a1 = array1
+        self._a2 = array2
+        #: The broadcast shape.
+        self.shape = broadcast.compute_broadcast_shape(array1.shape,
+                                                       array2.shape)
+
+    @staticmethod
+    def compute_broadcast_shape(shape1, shape2):
```

Comment:
:+1: - nice and simple. It will remove many of my tests too, so I'll go for that approach.